src/test/java/org/vafer/jdependency/ClazzpathUnitTestCase.java

```diff
@@ -69,10 +85,8 @@ public void testIssue47() throws IOException {
             .flatMap( i -> i.getClazzpathUnits().stream() )
             .map( i -> i.toString() )
             .collect(Collectors.toSet());
-        final Set<String> unitse = new HashSet<String>(Arrays.asList(
-            "woodstox-core-6.2.3.jar"
-            ));
-        assertEquals(unitse, units);
+        assertEquals(1, units.size());
+        assertTrue(units.iterator().next().endsWith("woodstox-core-6.2.3.jar"));
```

Comment:
Here I kind of prefer the original version. But I don't have my heart set on it.
Is this because it needs an `endsWith` now?

Reply:
Yes, the original code checks if both are equal which no longer works. In this code (probably because of the change in classloader) it had to change to the endsWith.

Reply:
Just curious - what does the prefix look like?

Reply:
It is the full path of the file on the system.  So this also changes between locally and in CI.

Reply:
I am wondering if this change might cause some incompatibilities for users then. For sure this deserves some consideration of at least a bigger version bump.

Reply:
This seems to be the only thing that still needs to resolved.

Is there a particular reason for the change of behaviour?
Maybe it makes more sense than the old behaviour?
Was this intentional?

Reply:
The actual code people use has not changed. Only the loading of the resources specifically for this test. On my machine the Set units actually contains this value `/home/nbasjes/workspace/Apache/jdependency/target/test-classes/woodstox-core-6.2.3.jar` ... on _my_ machine.

So instead of the pre filename (without path) my change in loading the resources for testing adds the entire local path and thus the compare needed to change.
Because there is only 1 value I chose this.

Reply:
OK, if this is only in the tests it's of course fine as is.